pkgs/development/tools/database/sqldeveloper/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, makeWrapper, requireFile, unzip, oraclejdk7}:
+
+stdenv.mkDerivation rec {
+  version = "4.0.3.16.84";
+  name = "sqldeveloper-${version}";
+
+  src = requireFile {
+    name = "${name}-no-jre.zip";
+    url = http://www.oracle.com/technetwork/developer-tools/sql-developer/downloads/index.html;
+    sha256 = "1qbqjkfda7xry716da2hdbbazks96rgyslrw1lw0azmqdp1mir7g";
+  };
+
+  buildInputs = [ makeWrapper unzip ];
+
+  buildCommand = ''
+    mkdir -p $out/bin
+    # patch to be able to install a sqldeveloper wrapper script compliant with nix's bin folder once installed
+    echo -e '#!/bin/bash\ncd "`dirname $0`"/../sqldeveloper/bin && bash sqldeveloper $*' >> $out/bin/sqldeveloper
```

Comment:
If I want to see sqldeveloper installed in PATH once installed in nix, it must be in a bin/ folder.
If I simply move the sqldeveloper.sh provided, it won't work because it hardcoded value. This is not free software so no contribution.

So I create an equivalent one with the .. path.
Also as a convenience, I removed the suffix `.sh`.

This `sqldeveloper` will be the one wrapped line 28.